docs/level-20.md

```diff
@@ -22,7 +22,7 @@ title: Level 20 - Ignition
 
 <br />
 
-### Double Ignition
```

Comment:
why did you change this

Reply:
okay I see - I changed this because it seems to be a parent section to both Double Ignition + Playful Ignition - both are types of 'double ignitions'. I guess we could also have the main category be 'Ignitions' and then separate them all out